### ISSUE TITLE
[Bug]: DN takes ~15 minutes to start logtail server after launch

### ISSUE THREAD

**Issue by aylei** [matrixorigin/matrixone#11983]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 1.0-dev-debug-42a457862
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

```
cat dn.log.txt | grep "MO logger init|start logtail"
{"level":"INFO","time":"2023/10/07 02:24:57.519199 +0000","caller":"logutil/internal.go:37","msg":"MO logger init, level=, log file=, stackLevel=panic"}
{"level":"INFO","time":"2023/10/07 02:40:01.756492 +0000","name":"tn-service.logtail-push-service","caller":"service/server.go:502","msg":"start logtail service","uuid":"36323365-3530-6635-6534-383637336132","server-id":"3393e41d-d19b-4df0-8bb7-a901fd97f382"}
```

### Expected Behavior

DN should start logtail server as soon as possible to keep the cluster in service, is 15 minutes too long?

### Additional information

Raw log: [dn.tgz](https://github.com/matrixorigin/matrixone/files/12836377/dn.tgz)

**Comment by aylei** [comment]
@jiangxinmeng1 please help take a look

**Comment by jiangxinmeng1** [comment]
fixed by #11988
